vmdb/app/models/ems_refresh/save_inventory_container.rb

```diff
@@ -51,7 +51,13 @@ def save_container_groups_inventory(ems, hashes, target = nil)
                 []
               end
 
-    save_inventory_multi(:container_groups, ems, hashes, deletes, [:ems_ref], [:container_definitions, :containers, :labels])
+    hashes.each do |h|
+      h[:container_node_id] = h.fetch_path(:container_node, :id)
```

Comment:
where is the id property modelled on a container_node?
there's uid (that goes to ems_ref) and name.